lib/tower_honeybadger/honeybadger/notice.ex

```diff
@@ -28,7 +28,7 @@ defmodule TowerHoneybadger.Honeybadger.Notice do
         stacktrace: stacktrace,
         plug_conn: plug_conn
       }) do
-    error_notice("(exit)", reason, stacktrace, plug_conn)
+    error_notice("(exit)", Exception.format_exit(reason), stacktrace, plug_conn)
```

Comment:
Moving this to a more generalized PR https://github.com/mimiquate/tower_honeybadger/pull/18.
This shouldn't be a Bandit specific change.